README.md

```diff
@@ -34,8 +34,8 @@ After checking out the sources from this repository, you need to create a
 [virtual environment](https://docs.python.org/3/tutorial/venv.html) and install the required packages:
 
 ```bash
-virtualenv -p python3 venv
-venv/bin/pip install -r requirements
+python3 -m venv venv
```

Comment:
This assumes that the module `venv` is installed, right? I tried this on a Ubuntu system today, and `virtualemnv` worked, whereas `python -m venv` did not.

Reply:
I have opposite situation. ```virtualenv -p python3 venv``` doesn't work for me on Ubuntu 16.04 whereas ```python3 -m venv venv``` is working. And on page https://docs.python.org/3/tutorial/venv.html you can see example:

"To create a virtual environment, decide upon a directory where you want to place it, and run the venv module as a script with the directory path"

```python3 -m venv tutorial-env```

Reply:
Ok, then there is apparently more than 1 way to do this...

Reply:
Ok. Please leave this as it is. Maybe add a comment below that some systems require `python3 -m venv venv` to create a virtual environment.